engine/class_modules/priest/sc_priest_shadow.cpp

```diff
@@ -316,6 +348,24 @@ struct mind_flay_t final : public priest_spell_t
     {
       priest().trigger_eternal_call_to_the_void( d );
     }
+
+    priest_td_t& td = get_td( d->target );
+    dot_t* swp      = td.dots.shadow_word_pain;
+    dot_t* vt       = td.dots.vampiric_touch;
+    dot_t* dp       = td.dots.devouring_plague;
+
+    int dots = swp->is_ticking() + vt->is_ticking() + dp->is_ticking();
+
+    /// TODO: Confirm is procrate is matching ingame
+    /// TODO: Confirm if this needs to be adjusted to be on its own PRNG system or has an ICD
+
+    if ( rng().roll( priest().specs.dark_thoughts->effectN( 1 ).percent() * dots ) )
+    {
+      // sim->print_log("{} rolled DT Proc success from flay with {} chance with {} dots", *player,
```

Comment:
can you swap this to just be a debug log? that way we can leave it in uncommented and still see it when debug is on

Reply:
Added to most recent commit, alongside of one for mind sear.